docs/user/prep-riboviz-operation.md

```diff
@@ -21,7 +21,15 @@ The workflow prepares ribosome profiling data for by implementing a workflow tha
 * `bam_to_h5.R`: convert BAM to compressed H5 format (local script, in `rscripts/`)
 * `generate_stats_figs.R`: generate summary statistics, analyses plots and QC plots (local script, in `rscripts/`)
 * `collate_tpms.R`: collate TPMs across samples (local script, in `rscripts/`)
-* `riboviz.tools.count_reads`: count the number of reads (sequences) processed by specific stages of the workflow (local script, in `riboviz/tools/`).
+* `riboviz.tools.count_reads`: count the number of reads (sequences) processed by specific stages of the workflow (local script, in `riboviz/tools/`).  
+* `AnalysisOutputs.Rmd`: the `staticHTML` Nextflow process runs an R markdown document which generates an HTML output report for each sample processed, using the analysed data (local script, in `rmarkdown/`)
+
+#### Interactive Visualization
+
+An additional script within the riboviz codebase allows the generation of interactive plots from the dataset after the riboviz workflow has been completed.  
+
+A process within the main riboviz workflow (`createInteractiveVizParamsConfigFile`, within `prep-riboviz.nf`) generates a configuration yaml file specifically for this interactive visualization (`interactive_viz_config.yaml`, found in the `dir_out` directory), and users can then run `rscripts/run_shiny_server.R` using this additional yaml (as detailed within the specific [run-shiny-server-operation.md documentation](./run-run_shiny_server-operation.md)).  This generates a shiny app instance which users can view to explore their data.
```

Comment:
riboviz => RiboViz 

```
as detailed within the specific [run-shiny-server-operation.md documentation](./run-run_shiny_server-operation.md)
```
=>
```
as detailed within [How To Run the Riboviz Interactive Data Visualization On Your Data](./run-run_shiny_server-operation.md)
```
 i.e. page title not page file name.

yaml => YAML

Reply:
Good catches, thanks